src/policies/domain_level_policy_blocks.py

```diff
@@ -30,11 +30,13 @@
 # ======================================================================================
 
 
-def reduce_work_models(contact_models, block_info, multiplier):
-    """Reduce contacts of workers by a multiplier.
+def reduce_work_models(
+    contact_models, block_info, attend_multiplier, hygiene_multiplier
+):
+    """Reduce contacts of workers by a attend_multiplier.
 
     Args:
-        multiplier (float or pd.Series): Must be smaller or equal to one. If a
+        attend_multiplier (float or pd.Series): Must be smaller or equal to one. If a
```

Comment:
`hygiene_multiplier` is missing in docstring

Reply:
Addressed in 8b56b1b